quill-core/src/main/scala/io/getquill/quotation/SchemaConfigParsing.scala

```diff
@@ -0,0 +1,33 @@
+package io.getquill.quotation
+
+import io.getquill.ast.PropertyAlias
+import scala.reflect.macros.whitebox.Context
+
+case class Config(alias: Option[String] = None, properties: List[PropertyAlias] = List(), generated: Option[String] = None)
+trait SchemaConfigParsing {
+  val c: Context
+  import c.universe.{ Function => _, Ident => _, _ }
+
+  def parseEntityConfig(t: Tree): Config =
+    t match {
+      case q"$e.entity(${ name: String })" =>
+        parseEntityConfig(e).copy(alias = Some(name))
+      case q"$e.columns(..$propertyAliases)" =>
+        parseEntityConfig(e).copy(properties = propertyAliases.map(parsePropertyAlias))
+      case q"$e.generated(($alias) => $body)" =>
+        parseEntityConfig(e).copy(generated = Some(parseProperty(body)))
+      case _ =>
+        Config()
+    }
+
+  private def parseProperty(t: Tree): String =
+    t match {
+      case q"$e.$property" => property.decodedName.toString
+    }
+
+  private def parsePropertyAlias(t: Tree): PropertyAlias =
+    t match {
+      case q"(($x1) => scala.this.Predef.ArrowAssoc[$t]($x2.$prop).->[$v](${ alias: String }))" =>
```

Comment:
@fwbrasil I tried to add unicode here, but the build failed. Do you know if the macro quasiquotation support unicode arrows?